docs/contributing/ui-text-style-guide.md

```diff
@@ -0,0 +1,15 @@
+# UI Text Style Guide
```

Comment:
Let's just add this to the existing contribution guide? This doesn't need to be a part of the main docs.